README.md

```diff
@@ -42,6 +42,12 @@ git clone https://github.com/nymanjens/quizmaster.git
 cd quizmaster
 docker-compose --file=docker-compose-build-locally.yml up
 ```
+- Run this command
+- This way, users can force Docker to recreate containers using the --force-recreate option.
+```
+- sudo docker-compose -f docker-compose-prebuilt.yml up --force-recreate
+```
+
```

Comment:
Wouldn't it make more sense to put this as a tip below the "Browse to http://localhost:9000/" bullet point? This is not really part of the installation guide.